### PR TITLE
Fix compatiblity with ppx_expect 0.14

### PR DESCRIPTION
**Pull request by rgrinberg**

### CODE REVIEW

**Comment by rgrinberg** [comment]
This should be mergeable once we have a version of ocamlformat that is compatible with base 0.14. @emillon any ETA no that?

File: .travis-ci.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ opam_install_test_deps () {
     opam install \
          ocamlfind \
          ppxlib \
-         ppx_expect.v0.13.1 \
+         ppx_expect \
```
Comment (jeremiedimino) [inline .travis-ci.sh:27]:
```suggestion
         'ppx_expect>=v0.14' \
```

`ppx_expect.config_types` doesn't exist in previous versions

**Comment by emillon** [comment]
@rgrinberg base 0.14 compat is merged in master (ocaml-ppx/ocamlformat#1396), we'll cut a release once 4.11 support is done. So a couple weeks I think. We can cut a point release if that takes too much time.

**Comment by rgrinberg** [comment]
Thanks. Ping me when the release is out if you don’t mind.

**Comment by emillon** [comment]
Sure! We've got open PRs for all remaining issues so it should just be about making and testing the release.

**Pull request merged**